src/components/DonutPath.js

```diff
@@ -0,0 +1,65 @@
+const React = require('react');
+const d3 = require('d3');
+
+const DonutPath = React.createClass({
+  propTypes: {
+    arc: React.PropTypes.func,
+    color: React.PropTypes.string,
+    opacity: React.PropTypes.number,
+    value: React.PropTypes.object
```

Comment:
AnimateOnLoad (which is added) has a default set at the DonutChart level. Same with opacity. Would these need another default value?

The arc and value should be required. The color isn't technically needed since D3 will default to black, but since there isn't a better way to default, I would say it should be required also.